### PR TITLE
Methods doc categories 

### PR DESCRIPTION
**Pull request by benjeffery**
I've had a go at subsections in the method list as it was getting unwieldy.

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -95,33 +98,69 @@ Methods
...
+------------------------------
+
+.. autosummary::
+   :toctree: generated/
+
    divergence
    diversity
    Fst
+   Tajimas_D
+   Weir_Goudet_beta
```
Comment (jeromekelleher) [inline docs/api.rst:114]: I think this does in the "pedigree and relatedness" bit?

File: docs/api.rst [hunk 2]
```diff
@@ -95,33 +98,69 @@ Methods
...
    Garud_H
-   genee
+   pbs
+
+Pedigree and Relationship
```
Comment (jeromekelleher) [inline docs/api.rst:118]:
```suggestion
Pedigree and Relatedness
```

File: docs/api.rst [hunk 3]
```diff
@@ -95,33 +98,69 @@ Methods
    count_cohort_alleles
    count_variant_alleles
    count_variant_genotypes
+   observed_heterozygosity
+   variant_stats
+
+Population and Cohort Genetics
```
Comment (jeromekelleher) [inline docs/api.rst:104]: I think this would be better as "Population genetic statistics"

File: docs/api.rst [hunk 4]
```diff
@@ -95,33 +98,69 @@ Methods
...
+   pedigree_contribution
+   pedigree_inbreeding
+   pedigree_inverse_kinship
+   pedigree_kinship
    hybrid_inverse_relationship
    hybrid_relationship
    identity_by_state
-   individual_heterozygosity
+
+Linkage Disequilibrium and Heterozygosity
```
Comment (jeromekelleher) [inline docs/api.rst:134]: I think just keep a section for LD, and move heterozygosity to "basic genetics" above (there's already one heterozygosity function above)

File: docs/api.rst [hunk 5]
```diff
@@ -86,6 +86,9 @@ Dataset
 Methods
 =======
 
+Allele and Genotype Analysis
```
Comment (jeromekelleher) [inline docs/api.rst:89]: Maybe "Basic genetics statistics", and move ``sample_stats`` up there?

**Review by jeromekelleher — COMMENTED** [review]
LGTM, minor reshuffle suggested

**Review by jeromekelleher — APPROVED** [review]
LGTM

**Comment by timothymillar** [comment]
LGTM aside from the above suggestion. The "Pedigree and Relatedness" section fells a little awkward, but I can't think of a better suggestion. I would almost suggest renaming it to just "Relatedness" or "Kinship and Relatedness". But it's not clear where `pedigree_contribution` would go.

File: docs/api.rst [hunk 6]
```diff
@@ -95,34 +98,69 @@ Methods
    count_cohort_alleles
    count_variant_alleles
    count_variant_genotypes
+   individual_heterozygosity
+   observed_heterozygosity
```
[the remark this answers is not in the document]
Comment (gregorgorjanc) [reply docs/api.rst:102]: Probably also `individual heterozygosity`?

**Comment by gregorgorjanc** [comment]
> LGTM aside from the above suggestion. The "Pedigree and Relatedness" section fells a little awkward, but I can't think of a better suggestion. I would almost suggest renaming it to just "Relatedness" or "Kinship and Relatedness". But it's not clear where `pedigree_contribution` would go.

Just “Relatedness” should cover it well - there are many facets behind that word

File: docs/api.rst (hunk 6 above)
[the remark this answers is not in the document]
Comment (timothymillar) [reply docs/api.rst:102]: It really depends how widely you want to cast the net. Currently, the pop-gen section only includes methods that generalize over 'cohorts' (and 'windows'). `individual_heterozygosity` is only for individuals and, in the diploid case, it just marks heterozygous and homozygous calls as 1 or 0 respectively.

**Comment by jeromekelleher** [comment]
OK, let's move heterozygosity to popgen for now, and rename the "pedigree.. " section to "Relatedness" and we're good to go I think.

**Comment by benjeffery** [comment]
Suggestions applied!

**Pull request merged**